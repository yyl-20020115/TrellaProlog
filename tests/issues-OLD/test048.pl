:-initialization(main).

main :-
	assertz('<https://josd.github.io/retina#p>'('<https://josd.github.io/retina#s>', '<https://josd.github.io/retina#o>')),
	'<https://josd.github.io/retina#p>'(S, O),
	writeq(S), nl, writeq(O), nl,
	writeq(ok), nl.
