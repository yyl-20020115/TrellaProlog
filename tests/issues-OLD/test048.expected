'<https://josd.github.io/retina#s>'
'<https://josd.github.io/retina#o>'
ok
